kuma/static/js/wiki-edit.js

```diff
@@ -544,6 +545,9 @@
     function saveDraft(val) {
         if (supportsLocalStorage) {
             localStorage.setItem(DRAFT_NAME, val || $form.find('textarea[name=content]').val());
+            var now = new Date();
+            var nowString = now.toLocaleDateString() + ' ' + now.toLocaleTimeString();
+            localStorage.setItem(DRAFT_NAME + '_SAVE_TIME', nowString);
```

Comment:
We're using the convention `DRAFT_NAME + '#save-time'`elsewhere on MDN, can you switch to that please?